### PR TITLE
fixes #72 : [copy] Review Club

### LINKED ISSUE #72: [copy] Review Club
The links for pre-req sections are not working. 

image

All the links will be same as this section of the readme. https://github.com/Bitshala/BitcoinCore-PR-Review-Club#prerequisites

### PR DESCRIPTION
**Pull request by keraliss**
Contributing to Bitcoin Core, a personal account
Onboarding to Bitcoin Core

these two links are not working. @rajarshimaitra  can you please take a look?

### CODE REVIEW

File: src/components/review/EssentialStudyMaterial.astro [hunk 1]
```diff
@@ -2,38 +2,38 @@
...
   },
   {
     name: "Introduction to Bitcoin Core Development",
-    link: "",
+    link: "https://bitcointechtalk.com/a-gentle-introduction-to-bitcoin-core-development-fdc95eaee6b8",
   },
   {
     name: "Contributing to Bitcoin Core, a personal account",
-    link: "",
+    link: "https://bitcointechtalk.com/contributing-to-bitcoin-core-a-personal-account-35f3a594340b",
```
Comment (rajarshimaitra) [inline src/components/review/EssentialStudyMaterial.astro:28]: New link

https://johnnewbery.com/contributing-to-bitcoin-core-a-personal-account/

File: src/components/review/EssentialStudyMaterial.astro [hunk 2]
```diff
@@ -2,38 +2,38 @@
...
   },
   {
     name: "Onboarding to Bitcoin Core",
-    link: "",
+    link: "https://medium.com/@amitiu/onboarding-to-bitcoin-core-7c1a83b20365",
```
Comment (rajarshimaitra) [inline src/components/review/EssentialStudyMaterial.astro:32]: This seems to be working for me..

File: src/components/review/EssentialStudyMaterial.astro [hunk 3]
```diff
@@ -77,9 +81,11 @@ const list2 = [
         <>
           <div class="mb-5 mt-10 flex justify-between lg:mr-10">
             <p class="font-black lg:text-xl">{item.name}</p>
-            <button class="w-1/5 rounded-lg border-2 border-dotted p-2 hover:bg-orange hover:text-white lg:w-auto lg:px-5 lg:py-2">
-              Read Now
-            </button>
+            <a href={item.link} target="_blank">
+              <button class="w-1/5 rounded-lg border-2 border-dotted p-2 hover:bg-orange hover:text-white lg:w-auto lg:px-5 lg:py-2">
```
Comment (emjshrx) [inline src/components/review/EssentialStudyMaterial.astro:85]: We dont need this button. The classes can be moved to the a tag itself

File: src/components/review/EssentialStudyMaterial.astro [hunk 4]
```diff
@@ -52,16 +52,20 @@ const list2 = [
...
-              >
-                View Code
-              </button>
+              <a href={item.link} target="_blank">
+                <button class="m-1 rounded-lg border-2 border-dotted hover:bg-orange hover:text-white lg:ml-2 lg:px-5 lg:py-2">
+                  Watch Video
+                </button>
+              </a>
+              <a href={item.code} target="_blank">
+                <button
```
Comment (emjshrx) [inline src/components/review/EssentialStudyMaterial.astro:61]: Dont need button

**Review by emjshrx — APPROVED** [review]

**Pull request merged**